src/main/resources/mcmod.info

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "modid": "mwc",
+    "name": "Modern Warfare Cubed",
+    "description": "Modern Warfare Cubed, VMW: Forked. PMW",
+    "version": "${version}",
+    "mcversion": "1.12.2",
+    "url": "",
+    "updateUrl": "",
+    "authorList": ["Panda"],
```

Comment:
Unsure but maybe @Paneedah would like to be referred as Paneedah, wait for his input on this though

Reply:
I'd prefer to be referred to as Paneedah, yes.

Reply:
Got it Paneedah